src/compiler-team.md

```diff
@@ -0,0 +1,106 @@
+# About the compiler team
+
+rustc is maintained by the
+[Rust compiler team](https://www.rust-lang.org/en-US/team.html). The
+people who belong to this team collectively work to track regressions
+and implement new features. Members of the Rust compiler team are
+people who have made significant contributions to rustc and its
+design.
+
+## Rust compiler meeting
+
+The compiler team has a weekly meeting where we do triage and try to
+generally stay on top of new bugs, regressions, and other things. This
+general plan for this meeting can be found in
+[the rust-compiler-meeting etherpad][etherpad]. It works roughly as
+follows:
+
+- **Review P-high bugs:** P-high bugs are those that are sufficiently
+  important for us to actively track progress. P-high bugs should
+  ideally always have an assignee.
+- **Look over new regressions:** we then look for new cases where the
+  compiler broke previously working code in the wild. Regressions are
+  almost always marked as P-high; the major exception would be bug
+  fixes (though even there we often
+  [aim to give warnings first][procedure]).
+- **Check I-nominated issues:** These are issues where feedback from
+  the team is desired.
+- **Check for beta nominations:** These are nominations of things to
+  backport to beta.
+
+The meeting currently takes place on Thursdays at 10am Boston time
+(UTC-4 typically, but daylight savings time sometimes makes things
+complicated).
+
+The meeting is held over a "chat medium" -- it used to be IRC, but we
+are currently in the process of evaluating other alternatives. Check
+[the etherpad] to find the current home (and see
```

Comment:
This link is `[the etherpad]` but the source below is `[etherpad]` so the link doesn't render.